tuftool/src/rhtas.rs

```diff
@@ -1010,11 +1021,33 @@ impl RhtasArgs {
         Ok(())
     }
 
-    fn load_target_bytes(target_path: &std::path::Path) -> io::Result<Vec<u8>> {
+    fn load_target_der_bytes(target_path: &Path) -> io::Result<Vec<Vec<u8>>> {
         let mut file = File::open(target_path)?;
-        let mut buffer = Vec::new();
-        file.read_to_end(&mut buffer)?;
-        Ok(buffer)
+        let mut buffer = String::new();
+        file.read_to_string(&mut buffer)?;
+
+        let mut result = Vec::new();
+        let mut content = String::new();
+        let mut inside_block = false;
+
+        for line in buffer.lines() {
+            if line.starts_with("-----BEGIN") {
+                content.clear();
+                inside_block = true;
+            } else if line.starts_with("-----END") {
+                let decoded = BASE64_STANDARD.decode(&content).map_err(|err| {
```

Comment:
It looks like we already have the https://docs.rs/x509-cert/latest/x509_cert/index.html library in our set of dependencies. Would it make more sense to use it to load the cert and decode it to pem instead of implementing it ourselves?